storage/simulation/default.script

```diff
@@ -0,0 +1,12 @@
+# operation[-value] first last every repeat
+# not all operations require a value
```

Comment:
yes, i read the parsing code. i call attention to this because it is unclear from this "doc" that it's **strictly** a single space between values